### PR TITLE
Adds the Everfull Mug ito tendril loot and also the wizard space ruins

### PR DESCRIPTION
**Pull request by SchrodingersWolf**
## What Does This PR Do
As title. Everfull Mugs have a chance to spawn in both the tendril loot pool and the wizard space ruin spawner.

## Why It's Good For The Game
The Everfull Mug is a really nice little piece of magic equipment - Not very powerful, but still has an effect that isn't entirely a meme. And yet somehow it is restricted to wizards only.

Adding this as a tendril loot to Mining makes sense mostly because Mining already gets a bunch of better healing and tendril loot is a bit lacking still. (If Balance would let me, I would like it to replace the cult bedsheets OR Hellwater OR Inferno Clusterbang spawn, those are drops that you are never happy to see)

As for the wiz ruin, it's a wizard ruin. It makes sense. (The owner of the ruin is also an IPC, so them leaving it behind in the engine room because it is useless to him makes sense)

All of the reagents inside the Everfull Mug also can already be acquired by crew means as well, so there would not be a extreme change in balance for crew use. (And heck, if miners donate the Everfull Mug to Sec during a cult or Vampire round, more power to them - After all, Chaplains can already mass bless water tanks anyways)

## Testing
Temporarily set necropolis chests to spawn only Everfull Mugs, killed a tendril, drank a bunch of ale to celebrate my victory.

Temporarily set wizard ruin spawner to spawn only Everfull Mug, killed the crate, drank a bunch of omnizine to heal my injuries.

## Changelog
:cl:
add: The wizard-exclusive Everfull Mug are now available in necropolis chests and the wizard space ruin.
/:cl:

### CODE REVIEW

File: code/modules/awaymissions/mission_code/ruins/wizardcrash.dm [hunk 1]
```diff
@@ -19,11 +19,12 @@
...
 				/obj/item/tarot_generator = 1,   // A little bit of everything, all of the time.
-				/obj/item/spellbook/oneuse/charge = 20,  // and now for less useful stuff to dilute the good loot chances
+				/obj/item/spellbook/oneuse/charge = 15,  // and now for less useful stuff to dilute the good loot chances
 				/obj/item/spellbook/oneuse/summonitem = 20,
 				/obj/item/spellbook/oneuse/forcewall = 10,
 				/obj/item/tarot_card_pack = 10,
 				/obj/item/tarot_card_pack/jumbo = 6,
 				/obj/item/tarot_card_pack/mega = 4,
-				/obj/item/book/granter/spell/summon_cheese = 15      // hungry wizard stuff
+				/obj/item/book/granter/spell/summon_cheese = 10      // hungry wizard stuff
```
Comment (Sirryan2002) [inline code/modules/awaymissions/mission_code/ruins/wizardcrash.dm:28]:
```suggestion
				/obj/item/book/granter/spell/summon_cheese = 10,      // hungry wizard stuff
```

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 2]
```diff
@@ -81,6 +81,8 @@
 					new /obj/item/tarot_card_pack/mega(src)
 				if(11)
 					new /obj/item/tarot_generator(src) // ~1/250? Seems reasonable
+		if(25)
```
Comment (Sirryan2002) [inline code/modules/mining/lavaland/necropolis_chests.dm:84]: this case is impossible with current code
`var/loot = rand(1, 23)`

**Review by Sirryan2002 — CHANGES_REQUESTED** [review]

File: code/modules/mining/lavaland/necropolis_chests.dm (hunk 2 above)
Comment (SchrodingersWolf) [reply code/modules/mining/lavaland/necropolis_chests.dm:84]: That is what I get for removing everything else to test whether it spawns or not

**Review by DGamerL — CHANGES_REQUESTED** [review]
Please apply the new loot range to the loot

File: code/modules/mining/lavaland/necropolis_chests.dm (hunk 2 above)
Comment (SchrodingersWolf) [reply code/modules/mining/lavaland/necropolis_chests.dm]: Loot range no longer a concern due to Everfull replacing Inferno clusterbang

**Review by Sirryan2002 — APPROVED** [review]

File: code/modules/awaymissions/mission_code/ruins/wizardcrash.dm [hunk 3]
```diff
@@ -19,11 +19,12 @@
...
 				/obj/item/tarot_generator = 1,   // A little bit of everything, all of the time.
-				/obj/item/spellbook/oneuse/charge = 20,  // and now for less useful stuff to dilute the good loot chances
+				/obj/item/spellbook/oneuse/charge = 15,  // and now for less useful stuff to dilute the good loot chances
 				/obj/item/spellbook/oneuse/summonitem = 20,
 				/obj/item/spellbook/oneuse/forcewall = 10,
 				/obj/item/tarot_card_pack = 10,
 				/obj/item/tarot_card_pack/jumbo = 6,
 				/obj/item/tarot_card_pack/mega = 4,
-				/obj/item/book/granter/spell/summon_cheese = 15      // hungry wizard stuff
+				/obj/item/book/granter/spell/summon_cheese = 10,      // hungry wizard stuff
```
Comment (DGamerL) [inline code/modules/awaymissions/mission_code/ruins/wizardcrash.dm:28]: Cheese nerf 😔

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 4]
```diff
@@ -81,6 +81,7 @@
 					new /obj/item/tarot_card_pack/mega(src)
 				if(11)
 					new /obj/item/tarot_generator(src) // ~1/250? Seems reasonable
+
```
Comment (DGamerL) [inline code/modules/mining/lavaland/necropolis_chests.dm:84]:
```suggestion
```

**Review by DGamerL — APPROVED** [review]

**Pull request merged**